### ISSUE TITLE
Standardization - Table Primary Key Naming

### ISSUE THREAD

**Issue by superdummy** [cviebrock/eloquent-taggable#22]
Hello, 

your library is cool (as written in another github issue), I use three of them. Thanks for developing it. 

Just a thought. Our world would be so much better if people agree more in things (often even disagreement leading to severe wars...). One small subset of "more agreeing" is standardization and yes the world would save billions of hours of work (that could then be used for other usefull stuff and making the world a better place), if stuff would be more standardized. 

Long story short: I personally would say that 	defining the primary Key of the Tag table as 
`$table->increments('tag_id');`
is not standard at all.
Standard would be: 
`$table->increments('id');`
This leads to various disadvantages like not remembering it and also to not be able to use highly standardized logic like "auto referenceing" tables based on convetions... 

Thanks!

**Comment by cviebrock** [comment]
Yeah, I get it.  There are advantages, however to using something like `tag_id` over just `id`, especially when joining many tables that all have an `id` column.

In any case, hopefully the package provides all the methods and services so the details are abstracted away.
